_scripts/clone-l10n.sh

```diff
@@ -1,64 +1,104 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
@jbuck is bash available on all our deploy boxes?

Reply:
Only need it when building the docker image, so yes